app/views/components/error_summary.scala.html

```diff
@@ -1,34 +0,0 @@
-@*
```

Comment:
@rhubner / @Stephen-Goddard Can you confirm that this component wasn't being used - i.e no error where being shown on the `Upload page`

Reply:
I thin it's not used anymore after we switched to new design.